tests/inputText.spec.ts

```diff
@@ -1,28 +1,39 @@
-import { chromium, test } from "@playwright/test";
+import { chromium, expect, test } from "@playwright/test";
 
-test("Typing a joke and pun should only appear on the page after clicking the Submit Joke button", async () => {
+test("Typing a joke and a pun and clicking the seperate buttons should render the text on the homepage", async () => {
```

Comment:
Much clearer description and pseudocode is very helpful to break down each step of the test as it passes 🥳